icons/cloud-alert.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "lscheibel"
```

Comment:
You should inherit the contributors from the icons you based this on.

Reply:
Done! Also added them to the cloud-check icon